Source/CBLDatabase+Replication.m

```diff
@@ -108,6 +108,26 @@ - (BOOL) putLocalCheckpointDocumentWithKey: (NSString*)key
     return result;
 }
 
+- (BOOL) removeLocalCheckpointDocumentWithKey: (NSString*)key
+                                     outError: (NSError**)outError {
+    if (key == nil)
+        return NO;
+    
+    NSMutableDictionary* document = [NSMutableDictionary dictionaryWithDictionary:
```

Comment:
This will crash if -getLocalCheckpointDocument returns nil. Instead you can use `[[[self getLocalCheckpointDocument] mutableCopy]`.